src/korg/fit.py

```diff
@@ -0,0 +1,41 @@
+from collections.abc import Iterable
```

Comment:
I think that you should rename this file as `_fit.py` to make it patently obvious that people should not try to directly import from this module.